docs/semgrep-app/notifications.md

```diff
@@ -277,31 +277,147 @@ Automated comments on GitLab merge requests are displayed as follows:
+
   rules:
+  # Scan changed files in MRs, (diff-aware scanning):
   - if: $CI_MERGE_REQUEST_IID
 
+  # Scan mainline (default) branches and report all findings.
+  - if: $CI_COMMIT_BRANCH == $CI_DEFAULT_BRANCH
+
   variables:
+    # Connect to Semgrep Cloud Platform through your SEMGREP_APP_TOKEN.
+    # Generate a token from Semgrep Cloud Platform > Settings
+    # and add it as a variable in your GitLab CI/CD project settings.
     SEMGREP_APP_TOKEN: $SEMGREP_APP_TOKEN
+    # Receive inline MR comments (requires Semgrep Cloud Platform account)
     GITLAB_TOKEN: $PAT
 ```
 
-For more config options, see [GitLab CI Sample](/semgrep-ci/sample-ci-configs/#gitlab-ci).
+Add the `SEMGREP_APP_TOKEN` to your GitLab repository by following these steps:
+
+1. Sign in to the [Semgrep Cloud Platform](https://semgrep.dev/), and then click the [Projects](https://semgrep.dev/orgs/-/projects) page.
+1. Select **Scan new project**, and then **Run Scan in CI**.
+1. Click **Create new API token**, and then copy the value of the new API token.
+1. In GitLab, add the token to your project by going to **Settings** > **CI/CD**.
+1. Under **Variables** click **Expand**, and then click **Add variable**.
+1. Enter `SEMGREP_APP_TOKEN` in the **Key** field and paste the token value in the **Value** field.
+1. Select **Mask variable** checkbox option, and then clear the **Protect variable** checkbox option.
+1. Click **Add variable**.
+
+For more configuration options, see [GitLab CI Sample](/semgrep-ci/sample-ci-configs/#gitlab-ci).
 
 :::note
-GitLab MR comments are only available to logged-in Semgrep users, as they require a Semgrep API token.
+GitLab MR comments are only available to logged-in Semgrep Cloud Platform users.
+:::
+
+### Bitbucket pull request comments
+
+To configure Semgrep PR comments in your Bitbucket PRs, follow the steps described in the subsections below.
+
+#### Creating a personal access token
+
+Create a personal access token to authenticate to the Bitbucket API. There are two ways in which you can create personal access tokens depending on the Bitbucket plan you use:
+
+- **Workspace access token**: If you use the Bitbucket Cloud Premium plan, you can create a workspace access token. This option saves time because you can create one access token for all repositories in the workspace. With one access token, you can bulk-onboard more repositories at once from a whole workspace. See [Creating a workspace access token](/semgrep-app/notifications/#creating-and-adding-a-workspace-access-token). However, you can also use the option of a repository access token, see below for more information to onboard repositories one by one.
+- **Repository access token**: If you are **not** using the Bitbucket Cloud Premium plan, you have to create a separate repository access token for each repository where you want to use Semgrep. See [Creating a repository access token](/semgrep-app/notifications/#creating-and-adding-a-repository-access-token).
```

Comment:
```suggestion
- **Repository access token**: If you do **not** have the Bitbucket Cloud Premium plan, create a separate repository access token for each repository where you want to use Semgrep. This configuration option is also useful if you have the Bitbucket Cloud Premium plan, but prefer to onboard repositories one by one instead of bulk onboarding. See [Creating a repository access token](/semgrep-app/notifications/#creating-and-adding-a-repository-access-token).
```